stripe/api_version.py

```diff
@@ -2,5 +2,5 @@
 # File generated from our OpenAPI spec
 
 
-class _ApiVersion:
-    CURRENT = "2022-11-15"
+class ApiVersion:
```

Comment:
Changes _ApiVersion to ApiVersion, as I think it's legitimate for users to want to ask what the pinned version of the library is.

They have `Stripe.api_version`  for that.

Reply:
Stripe.api_version is the version *currently in use*. It *defaults* to ApiVersion.CURRENT, but is not a constant describing the library's pinned version.

Reply:
Also FWIW this is public in every other library.

Reply:
We also don't have a similar feature in node.